config/prod/htan-dcc-center-a.yaml

```diff
@@ -19,19 +19,17 @@ parameters:
     - "3413795"
   S3UserARNs:
     - "arn:aws:iam::070278699608:user/inodb"
-    - "arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jason.hwee@sagebase.org"
-    - "arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/adam.taylor@sagebase.org"
+    - "arn:aws:sts::055273631518:assumed-role/AWSReservedSSO_viewer-plus_96e9ce963c940f21/adam.taylor@sagebase.org"
```

Comment:
adam didn't actually have access to this profile.  I just gave him access in Jumpcloud though.

Reply:
@BrunoGrandePhD I'm not sure I want to make scicomp an AWS user account store.  I'm wondering whether it would be more appropriate to give @adamjtaylor access to one of your project accounts then use the user ARN of that project account here?

Reply:
[IT-1967](https://sagebionetworks.jira.com/browse/IT-1967?focusedCommentId=148311) was created to figure this out, but no one from IT has responded in over a month. I made my suggestion there, but it's up to IT to figure out a transition plan away from sandbox roles. 

We don't yet have a production-grade project AWS account for HTAN, and I don't have the bandwidth to create a new AWS account just for housing an IAM role. 

I don't want this to hold up this PR, so I reverted these role changes. This PR is now solely for updating the stack version. We can update the role ARNs once IT-1967 is settled.

Reply:
I never said i was completely opposed to using @adamjtaylor scicomp here @BrunoGrandePhD.  It was really just a question and suggestion.   It's your choice to revert but i'm amendable to keeping his scicomp account ARN here if there's not a better option at this time.

Reply:
I never said that you were completely opposed. I just don't know how long that conversation will take, and I need to debug an issue for a collaborator, who has been waiting since last week. So, I'm uncoupling IT-1967 from this PR, which was primarily motivated to unblock me so I can help our collaborator. 

Please merge if you approve this PR and we'll deal with IT-1967 separately.